src/wam2layers/utils/profiling.py

```diff
@@ -109,37 +109,44 @@ def track_stability_correction(
 
         Criteria:
         1. A correction is applied for more than 5% of the grid (first time)
-        2. Previous correction is exceeded by either 5%-point of grid
-        3. Previous correction is doubled in magnitude
+        2. At least 10% more of the grid is corrected compared to the last warning
```

Comment:
Isn't that quite often? So if I first get a warning at 5%, I will get the next one at 5.5%, and then at 6.05%, etc. I would make it at least 50% if relative, or stick to using increases in %-points

Reply:
A few more warnings and 2D-grids to analyze are not so problematic, but rather informative, I suppose. The main issue was, however, to provide more informative on-screen logging. See #280